avalanche/benchmarks/utils/detection_dataset.py

```diff
@@ -660,7 +472,7 @@ def concat_detection_datasets(
         Sequence[TTargetType], Sequence[Sequence[TTargetType]]
     ]] = None,
     collate_fn: Optional[Callable[[List], Any]] = None
-) -> Union[DetectionDataset, SupervisedDetectionDataset]:
+) -> DetectionDataset:
```

Comment:
do we need this? DetectionDataset should be enough. We have the others for classificaiton mostly to keep it compatible with the old API.